node/lib/package/LocalPackageSource.ts

```diff
@@ -303,6 +305,7 @@ export class LocalPackageSource extends PackageSource {
                 let onboardingPackageName: string =
                         LocalPackageSource.derivePackageName(onboardingId, "onboarding");
                 onboardingModulePath = onboardingPackageName + "/" + onboardingId;
+
```

Comment:
Forgot the `/js/` directory.

---
In reply to: [75516149](https://github.com/openT2T/opent2t/pull/22#discussion_r75516149) [](ancestors = 75516149,75431448)